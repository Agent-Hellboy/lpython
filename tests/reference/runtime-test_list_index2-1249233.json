{
    "basename": "runtime-test_list_index2-1249233",
    "cmd": "lpython {infile}",
    "infile": "tests/errors/test_list_index2.py",
    "infile_hash": "f7f556f1c048a88ee6eb9eae3ce3a337da17b3705b602085ff62ac4f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "runtime-test_list_index2-1249233.stderr",
    "stderr_hash": "5ea5ede0ac7cba17b606986071bab9c53025ce7ab45fbf0ac857b2ef",
    "returncode": 1
}