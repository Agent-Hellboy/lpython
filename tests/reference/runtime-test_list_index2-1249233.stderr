ld: warning: object file (test_list_index2.out.tmp.o) was built for newer macOS version (14.0) than being linked (13.3)
ValueError: The list does not contain the element: 1
